ffw/NavigationRPC.js

```diff
@@ -502,6 +502,11 @@ FFW.Navigation = FFW.RPCObserver.create(
         return;
       }
 
+      // (&& !params) - params take precedent over info
+      if (info && resultCode === SDL.SDLModel.data.resultCode.WARNINGS) {
```

Comment:
Actually it is applicable for any successful result code
```suggestion
      if (info && is_successful_code) {
```